website/docs/docs/cloud/manage-access/environment-permissions-setup.md

```diff
@@ -15,33 +15,34 @@ Environment-level permissions are not the same as account-level [role-based acce
 
 In your dbt Cloud account:
 
-1. Click your account name, above your profile icon on the left side panel, then select **Account settings**. From there, select **Groups & Licenses**. While you can edit existing groups, we recommend not altering the default `Everyone`, `Member`, and `Owner` groups.
+1. Click your account name, above your profile icon on the left side panel, then select **Account settings**.
+2. From there, select **Groups & Licenses**. We recommend deleting the default `Owner`, `Member`, and `Everyone` groups before deploying and replacing them with your organizational groups. 
```

Comment:
```suggestion
2. Select **Groups & Licenses**. We recommend deleting the default `Owner`, `Member`, and `Everyone` groups before deploying and replacing them with your organizational groups to avoid granting users unnecessary elevated privileges.
```